src/Utils.php

```diff
@@ -374,10 +374,10 @@ private static function idnToAsci(string $domain, int $options, ?array &$info =
             return $domain;
         }
```

Comment:
Do we need the regexe? Since we convert by default to `false` it's the users problem to deal with...